types/passport-facebook/index.d.ts

```diff
@@ -1,66 +1,102 @@
-// Type definitions for passport-facebook 2.1
+// Type definitions for passport-facebook 3.0
 // Project: https://github.com/jaredhanson/passport-facebook
 // Definitions by: James Roland Cabresos <https://github.com/staticfunction>, Lucas Acosta <https://github.com/lucasmacosta>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import passport = require('passport');
-import express = require('express');
-
-export interface Profile extends passport.Profile {
-    id: string;
-    displayName: string;
-    gender?: string | undefined;
-    ageRange?: {
-        min: number;
-        max?: number | undefined;
-    } | undefined;
-    profileUrl?: string | undefined;
-    username?: string | undefined;
-    birthday: string;
-
-    _raw: string;
-    _json: any;
-}
+import * as passport from 'passport';
+import * as express from 'express';
+import * as oauth2 from 'passport-oauth2';
+import { OutgoingHttpHeaders } from 'http';
+
+import facebook = Strategy;
+
+declare class Strategy extends oauth2.Strategy {
+    constructor(options: facebook.StrategyOptions, verify: facebook.VerifyFunction);
+    constructor(options: facebook.StrategyOptionsWithRequest, verify: facebook.VerifyFunctionWithRequest);
 
-export interface AuthenticateOptions extends passport.AuthenticateOptions {
-    authType?: string | undefined;
+    name: string;
+    authenticate(req: express.Request, options?: object): void;
+    authorizationParams(options: facebook.AuthorizationParamsOptions): object;
 }
 
-export interface StrategyOption {
-    clientID: string;
-    clientSecret: string;
-    callbackURL: string;
+declare namespace Strategy {
+    // NOTE: not true for `export import` statements
+    // tslint:disable-next-line:strict-export-declare-modifiers
```

Comment:
Fix this lint:
Instead of moving these interfaces and classes into namespace, pls pull them out with direct export like before.

Reply:
hi @sheetalkamat, I attempted to fix this issue in [fb28d1e](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65543/commits/fb28d1e62101fd68028a9b8e25d0d84c2d6b39f4). Is there something else I need to do to appease @DangerBotOSS?